### PR TITLE
Intro to r script

### PR DESCRIPTION
**Pull request by benmbrew**
Still not finished, but when you get chance (Joe), could you look over what I've done. 

Also, do you want to tackle the sections i still need to do (dplyr, statistical analysis)? I feel like you might know dpylr a little better.  I'll continue writing instructions for installing R, R studio and the packages required for the tutorial.

### CODE REVIEW

**Comment by benmbrew** [comment]
Do you see it now?

File: R/intro_to_R/installing_R.Rmd [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (joebrew) [inline R/intro_to_R/installing_R.Rmd:1]: @benmbrew Why is this markdown here? It seems like it's not going to be used. Delete.

**Comment by joebrew** [comment]
@benmbrew Two things:

1. We shouldn't track "binary" files (`.rda` or `.RData`) in github. Please remove them from this PR.

2. There is a `.Rmd` with nothing on it. Please remove that as well.

Ping me once you've done these two things.

**Comment by joebrew** [comment]
@benmbrew I'm going to merge this, then move things around substantially. I'll ping you later.

**Pull request merged**